Makefile

```diff
@@ -96,12 +90,8 @@ build-release-tarballs:
 	[ -d $(BIN_DIR) ] || mkdir -p $(BIN_DIR)
 	$(MAKE) build-x86_64
 	$(call tarball_release_binary,$(BUILD_PATH_X86_64),$(X86_64_TAG),"")
-	$(MAKE) build-x86_64-portable
-	$(call tarball_release_binary,$(BUILD_PATH_X86_64),$(X86_64_TAG),"-portable")
```

Comment:
Not sure if I can delete this and the below `portable` one

Reply:
Looks ok to me!